file/writer_test.go

```diff
@@ -341,3 +347,44 @@ services:
 }`
 	assert.Equal(expected, output)
 }
+
+func Test_getFormatVersion(t *testing.T) {
+	tests := []struct {
+		name        string
+		kongVersion string
+		expected    string
+		expectedErr string
+		wantErr     bool
+	}{
+		{
+			name:        "3.0.0 version",
+			kongVersion: "3.0.0",
+			expected:    "3.0",
+		},
+		{
+			name:        "2.8.0 version",
+			kongVersion: "2.8.0",
+			expected:    "1.1",
+		},
```

Comment:
Please add test for 4 digit versions and add test for versions with a suffix string (like 2.8.0.1-enterprise-edition or something like that).

Reply:
https://github.com/Kong/deck/pull/753/commits/e5d3912bd1141f0fec3d4694bd8272942862aa8d